hound_test.rb

```diff
@@ -0,0 +1,7 @@
+name = 'greg'
+
+puts name 
+puts 'something'
+puts 'hello world'
+puts 'another violation'
+puts 'another violation 2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.